data-anonymization/presdio/README.md

```diff
@@ -0,0 +1,210 @@
+
+![bacalhau-data-anonynization.png](img/bacalhau-data-anonynization.png)
+
+### Regional Data Sovereignty
+
+The architecture maintains strict data boundaries by implementing separate storage and compute resources in different geographical regions. In the European region, we maintain a secure MinIO storage bucket that houses the confidential data, paired with dedicated compute nodes running Microsoft Presidio for data anonymization. This setup ensures that sensitive data remains within EU boundaries at all times, complying with data protection regulations.
+
+A parallel infrastructure exists in the US region, consisting of its own MinIO storage bucket and compute resources. However, this region only receives and processes already anonymized data, ensuring compliance with cross-border data transfer regulations.
+
+### Orchestration and Control
+
+At the heart of our architecture lies the Bacalhau orchestrator node, which coordinates the entire data processing workflow. The orchestrator manages job distribution, resource allocation, and workflow coordination without ever handling the sensitive data directly. This separation of control plane and data plane is crucial for maintaining security and compliance.
+
+### Distributed Processing
+
+The compute layer consists of multiple worker nodes deployed in the EU region, each capable of running Microsoft Presidio for data anonymization. These nodes operate in parallel, processing different portions of the confidential data simultaneously. This distributed approach not only improves processing efficiency but also maintains data locality, ensuring that sensitive information never leaves its designated security boundary.
+
+### Secure Data Flow
+
+The data flow follows a strict pattern:
+1. Confidential data resides in the EU region's MinIO bucket
+2. EU-based compute nodes process and anonymize the data using Microsoft Presidio
+3. Only anonymized results are transferred to the US region's MinIO bucket
+4. US-based compute nodes can then perform analytics on the sanitized data
+
+This architecture demonstrates how organizations can leverage distributed computing to maintain compliance with data protection regulations while enabling global data analytics capabilities. The solution is packaged as a self-contained demonstration environment using Docker Compose, making it easy to deploy and test in any environment.
+
+
+# Running the Demo Environment
+
+## System Requirements
+
+Before starting, ensure your system meets the following minimum requirements:
+- 20 GB of free disk space
+- 4 CPU cores
+- Docker and Docker Compose installed
+
+## Quick Start
+
+### Prerequisites
+- Docker and Docker Compose
+- Access to example multi-region setup
+
+### Deploy Environment
+```bash
+# Clone and start the multi-region environment
+git clone https://github.com/bacalhau-project/examples.git
+cd examples/setting-up-bacalhau-cluster/docker-compose/multi-region
+docker compose up -d
```

Comment:
Heads up that I moved the network setup to a new dedicated repo
https://github.com/bacalhau-project/bacalhau-network-setups

The intention to have a common place for different ways to setup bacalhau network. For now I just moved docker-compose, and will review the other terraform options in this repo, but I feel they are very opinionated

Reply:
Fixed.